vue/components/ui/organisms/form/form.vue

```diff
@@ -147,6 +159,18 @@ body.mobile .form > .form-form > .tabs .column {
 .form > .form-form > .tabs .column > .section > .section-field:last-child {
     margin-bottom: 0px;
 }
+
+.form > .form-form > .tabs .column > .section > .section-field ::v-deep .content {

```

Comment:
Which is the component whose content alignment you're trying to override? Shouldn't it be a prop of that component and not a v-deep?